runtime/file.cpp

```diff
@@ -22,6 +28,35 @@ void OpenFile::set_path(OwningPtr<char> &&path, std::size_t bytes) {
   pathLength_ = bytes;
 }
 
+static int openfile_mkstemp(IoErrorHandler &handler) {
+#ifdef _WIN32
+  const unsigned int uUnique{0};
+  char tempDirName[MAX_PATH-14];
```

Comment:
Why the adjustment by 14 here and below?  Is this a magic constant that should be named?

Reply:
The input to GetTempFileNameA should be less than or equal to MAX_PATH-14. Docs at https://docs.microsoft.com/en-us/windows/win32/api/fileapi/nf-fileapi-gettempfilenamea

Reply:
A comment to that effect would reduce the mystery.